controller/src/main/java/org/apache/rocketmq/controller/impl/manager/ReplicasInfoManager.java

```diff
@@ -387,6 +429,29 @@ private void handleElectMaster(final ElectMasterEvent event) {
         }
     }
 
+    private void handleCleanBrokerDataEvent(final CleanBrokerDataEvent event) {
+        final String brokerName = event.getBrokerName();
+        final Set<String> brokerAddressSet = event.getBrokerAddressSet();
+
+        if (!isContainsBroker(brokerName) || null == brokerAddressSet || brokerAddressSet.isEmpty()) {
```

Comment:
Why do we need `this.replicaInfoTable.remove(brokerName)` and `this.syncStateSetInfoTable.remove(brokerName)` when !isContainsBroker(brokerName)?